pom.xml

```diff
@@ -751,6 +753,49 @@
           <argLine>-Xms256m -Xmx512m</argLine>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-pmd-plugin</artifactId>
+        <configuration>
+          <targetJdk>${java.version}</targetJdk>
+          <!-- TODO: Flip this to true once violations are evaluated or fixed -->
+          <failOnViolation>false</failOnViolation>
```

Comment:
How many violations are there? Is it small enough it could be posted in a gist or something?

Reply:
From the travis build it looks like 11 violations.

[Gist of the entire travis build for convenience/history](https://gist.github.com/jhmcstanton/8e797f967ac3f85d998c034b902dbf8e)

The beginning of the gist contains the violations

Reply:
Thanks. These look easy to fix. I logged issue #122 to fix them.